cookbooks/bcpc-hadoop/templates/default/hb_hbase-site.xml.erb

```diff
@@ -166,4 +156,143 @@
     <value>org.apache.hadoop.hbase.ipc.SecureRpcEngine</value>    
   </property>   
   <% end %>
+  
+  <% if node["bcpc"]["hadoop"]["hbase"]["shortcircuit"]["read"] == true then %>
+  <property>
+    <name>dfs.client.read.shortcircuit</name>
+    <value><%= node["bcpc"]["hadoop"]["hbase"]["shortcircuit"]["read"]%></value>
+  </property>
+
+  <property>
+    <name>dfs.domain.socket.path</name>
+    <value>/var/run/hadoop-hdfs/dn._PORT</value>
+  </property>
+
+  <property>
+    <name>dfs.client.read.shortcircuit.buffer.size</name>
+    <value>131072</value>
+  </property>
+  <% else %>
+  <property>
```

Comment:
Why not move this outside of the conditional than repeat the code here and above for `dfs.client.read.shortcircuit`?